<launch>
  <include file="$(find wheeled_robin_bringup)/launch/minimal_fake.launch" />
  <include file="$(find turtlebot_bringup)/launch/3dsensor.launch">
    <arg name="rgb_processing" value="true"/>
    <arg name="depth_processing" value="true"/>
    <arg name="depth_registered_processing" value="false"/>
    <arg name="depth_registration" value="false"/>
    <arg name="disparity_processing" value="false"/>
    <arg name="disparity_registered_processing" value="false"/>
    <arg name="scan_processing" value="false"/>
  </include>
  <node name="ar_kinect" pkg="ar_kinect" type="ar_kinect" respawn="false" output="screen">
    <remap from="points" to="/camera/depth_registered/points"/>
    <param name="marker_pattern_list" type="string" value="$(find wheeled_robin_formation_drive)/data/objects_kinect"/>
    <param name="marker_data_directory" type="string" value="$(find ar_pose)"/>
    <param name="threshold" type="int" value="100"/>
    <param name="publish_tf" type="bool" value="true" />
  </node>
  <param name="goal_frame" type="string" value="goal_pattern"/>	
  <node name="marker_validation" pkg="wheeled_robin_formation_drive" type="marker_validation" >
    <param name="marker_frame" type="string" value="master_pattern"/>
    <param name="x_offset" type="double" value="1.0"/>
    <param name="y_offset" type="double" value="0.0"/>
  </node>
  <node name="calc_vel" pkg="wheeled_robin_formation_drive" type="vel_control" >
    <param name="p_lin" type="double" value="4.0"/>
    <param name="p_ang" type="double" value="0.5"/>
  </node>
</launch>
